lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1934,7 +1934,7 @@ def test_denied_email_message_with_valid_due_date_and_attempts_allowed(self):
             "We could not verify your identity for the {assessment} assessment "
             "in the {course_name} course. You have used "
             "{used_attempts} out of {allowed_attempts} attempts to "
-            "verify your identity.".format(
+            "verify your identity".format(
```

Comment:
This has been fixed on the release, so it won't be merged back until that goes out.